### DISCUSSION TITLE
How to get access to Anker SolixAPI?

### DISCUSSION THREAD

**Discussion by amor20git** [thomluther/anker-solix-api#133] [General]
I am searching the internet and Anker homepage all day long but I am not successful: Where can I find the access to SolixAPI and get the API keys?

**Comment by thomluther** [comment]
@amor20git You can use this library and the example to create a Api login for your test client.
You have to review the source code in request.py how manual Api requests are build.
Easier would be to use a Api client like in the example, then you don't have to care about the keys.
Each client connection will create a new Login request if the login response is not locally cashed.
If the token of the cached response is no longer available on the server, the library automatically creates a new login request to refresh and cache the new token (as well as the whole login response)

The token is valid until another client starts a new login request, since the server allows only 1 active token.
